ocs_ci/utility/uninstall_openshift_logging.py

```diff
@@ -0,0 +1,52 @@
+"""
+Function to teardown the openshift-logging
+"""
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs import ocp
+
+logger = logging.getLogger(__name__)
+
+
+def uninstall_cluster_logging():
+    """
+    Function to uninstall cluster-logging from the cluster
+    1. Deletes the project "openshift-logging"
+        & "openshift-operators-redhat"
+    2. Deletes the CSC resource "cluster-logging-operator"
+        & "elasticsearch"
+    """
+    namespace_obj1 = ocp.OCP(
+        kind='Namespaces', namespace='openshift-logging'
+    )
+    namespace_obj2 = ocp.OCP(
+        kind='Namespaces', namespace='openshift-operators-redhat'
+    )
+
+    if namespace_obj1.get():
+        logger.info("The namespaces openshift-logging exists, Deleting.......")
+        assert namespace_obj1.delete(resource_name='openshift-logging')
+        logger.info("The project openshift-logging got deleted successfully")
+    else:
+        logger.error("The project openshift-logging does not exists")
```

Comment:
Please use this documented procedure to uninstall logging before deleting the project - https://access.redhat.com/documentation/en-us/openshift_container_platform/4.1/html-single/logging/index#efk-logging-uninstall

Reply:
Done